seatunnel-connectors-v2/connector-paimon/src/main/java/org/apache/seatunnel/connectors/seatunnel/paimon/catalog/PaimonCatalog.java

```diff
@@ -161,6 +164,25 @@ public void createTable(TablePath tablePath, CatalogTable table, boolean ignoreI
             throw new TableAlreadyExistException(this.catalogName, tablePath);
         } catch (org.apache.paimon.catalog.Catalog.DatabaseNotExistException e) {
             throw new DatabaseNotExistException(this.catalogName, tablePath.getDatabaseName());
+        } catch (Exception e) {
```

Comment:
The suggestion is to extract a method to handle the exception, which is weird and not very readable.